src/action_management/include/actionmanagement.h

```diff
@@ -0,0 +1,339 @@
+#ifndef _ACTIONS_H_
+#define _ACTIONS_H_ 
+
+#include <stdio.h>
+
+
+/* An enumeration of all supported actions. */
+/* All synonyms will be provided to CLI. */
+enum actions {
+  OPEN, /* world action */
+  CLOSE, /* world action */
+  PUSH, /* world action */
```

Comment:
Would this be like:
```suggestion
  PUSH, /*synonymous with "use" but with only one argument, world action */
```
I'm thinking of like pull lever push button

Reply:
I think we were leaning more toward "use" as using items from inventory.